### PR TITLE
Fix SDK-648

### PR DESCRIPTION
**Pull request by jpcostello**
Fix ** User is able to complete the transaction when "Card holders name" and CVV is not entered in the card widget for WooCommerce

### CODE REVIEW

File: resources/js/frontend/index.js [hunk 1]
```diff
@@ -31,22 +29,22 @@ const label = decodeEntities(settings.title) || labels.defaultLabel;
...
 
     useEffect(() => {
-        let newAmount = null;
-
-        const unsubscribeFromShippingEvent = onShippingRateSelectSuccess(async () => {
-            const { total_price: currentTotalPrice } = cart.getCartTotals();
-
-            newAmount = Number(currentTotalPrice / 100).toFixed(2);
-        });
-
```
Comment (jpcostello) [inline resources/js/frontend/index.js:46]: @andryes @FabianaRBarros @Danyl-Kushnaryov This code above was a fix for something else?

Comment (Danyl-Kushnaryov) [reply resources/js/frontend/index.js:46]: @jpcostello @anthonyvallone-paydock  All code written to fix this bug: https://paydock.atlassian.net/browse/SDK-592, has been removed in this PR

Comment (jpcostello) [reply resources/js/frontend/index.js:46]: @anthonyvallone-paydock I raised this PR from your branch - some code may still need to be merged based on comments. Please take a look.

Comment (anthonyvallone-paydock) [reply resources/js/frontend/index.js:46]: @Danyl-Kushnaryov i just branched off the most up to date power_board branch and made my changes.

Comment (Danyl-Kushnaryov) [reply resources/js/frontend/index.js:46]: The removed part above is present in the power_board branch:

https://github.com/PayDock/e-commerce-woo/blob/power_board/resources/js/frontend/index.js#L39-L45

**Comment by anthonyvallone-paydock** [comment]
I know what the issue was @Danyl-Kushnaryov 

I was working off the gitlab saas repo when i originally made my changes. I realised you guys are working off github and i copy pasted before i pushed and missed some changes in github vs gitlab.

Ive amended my branch

**Review by andryes — APPROVED** [review]

**Review by andryes — APPROVED** [review]

**Pull request merged**